policy-engine/src/config/settings.rs

```diff
@@ -61,12 +67,28 @@ impl AppSettings {
         // Combine options into a single config.
         let mut cfg = defaults;
         cfg.merge(cli_opts);
-        cfg.merge(file_opts);
+        cfg.try_merge(file_opts)?;
 
         // Validate and convert to settings.
         Self::try_validate(cfg)
     }
 
+    /// Validate and return policy plugins.
+    pub fn policy_plugins(&self) -> Fallible<Vec<BoxedPlugin>> {
+        let mut configured_plugins = Vec::with_capacity(self.policies.len());
+        for conf in &self.policies {
+            let plugin = conf.build_plugin()?;
+            configured_plugins.push(plugin);
+        }
+
+        // TODO(lucab): drop this as soon as all config-maps are in place (prod & staging).
+        if configured_plugins.is_empty() {
```

Comment:
This PR seemed big enough already, so I'm trying not to entangle it with prod+staging config bumping.
Thus, policy-engine still fallbacks to the hardcoded config for now, until we specificy (and populate) a TOML file via ConfigMap in both environments.
Let me know if you want to proceed otherwise.

Reply:
I'm fine with that. I would be even more fine if we had an integration test which provides a TOML file and verifies the system as close production as possible.

Reply:
Right now I'm hardcoding the default plugins in `run_test` here: https://github.com/openshift/cincinnati/pull/129/files#diff-ffbd2fce11fad2ac24f3974e0e763228R334.

Do you want me to use a TOML config in there instead?

Reply:
> Do you want me to use a TOML config in there instead?

I'd prefer that so it would cover the read-from-file code too, right?

Reply:
Sure, a TOML config read from a file. I was more asking about re-using that test. It seems like it'll be fine.

Reply:
You lost me there, would you mind to elaborate?

Reply:
Clarified out-of-band, we were stating the same thing in different ways.